### PR TITLE
Zb dinos

### PR DESCRIPTION
**Pull request by zbright**
@zsyed1 - Thanks for all the feedback! I made your suggested changes, if there is anything else you think could use some work just let me know

### CODE REVIEW

File: dino_catalog/catalog.rb [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+            "Bipeds - Display all bipeds",
+            "Carnivores - Display all carnivores",
+            "Time Period - Display all from a specified time period",
+            "Big - Display all big dinos",
+            "Small - Display all small dinos",
+            "Select - Select one dion to view",
+            "Reset - Remove all filter",
+            "Exit - quit program",
+            "You can also hit enter to view current list again"].join("\n")
+
```
Comment (zsyed1) [inline dino_catalog/catalog.rb]: I would put this string (lines17 - 26) into a method `def menu` but this is fine too.

File: dino_catalog/catalog.rb [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    end
+
+    dino_list.get_big if user_input.include?('big')
+    dino_list.get_small if user_input.include?('small')
+
+    if user_input.include?('select')
+      puts "Please enter a dinosaur's name/genus: "
+      name_input = $stdin.gets.chomp.downcase
+
+      dino_list.get_by_name(name_input)
```
Comment (zsyed1) [inline dino_catalog/catalog.rb]: No need for an empty line (line 55)

File: dino_catalog/dino_list.rb [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+      arr << dino.walking
+      arr << dino.description if print_desc
+      arr
+    end
+  end
+
+  def to_json
+    dinos = []
+    each { |dino| dinos << dino.to_json }
+
```
Comment (zsyed1) [inline dino_catalog/dino_list.rb]: You can do this with a `map` call: `puts map { |dino| dino.to_json }`

File: dino_catalog/main.rb [hunk 4]
```diff
@@ -0,0 +1,5 @@
+require 'csv'
+require 'terminal-table'
+require_relative 'catalog'
+
+Catalog.new(ARGV[0]).run
```
Comment (zsyed1) [inline dino_catalog/main.rb]: Nice work.

File: dino_catalog/catalog.rb [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+            "Time Period - Display all from a specified time period",
+            "Big - Display all big dinos",
+            "Small - Display all small dinos",
+            "Select - Select one dion to view",
+            "Reset - Remove all filter",
+            "Exit - quit program",
+            "You can also hit enter to view current list again"].join("\n")
+
+      user_input = $stdin.gets.chomp.downcase
+      done = true and next if user_input == 'exit'
```
Comment (jmmastey) [inline dino_catalog/catalog.rb]: Or just break out of the loop. `while true; break if user_input == exit`

**Pull request closed without merging**